tools/c7n_azure/c7n_azure/storage_utils.py

```diff
@@ -101,7 +108,9 @@ def create_queue_from_storage_account(storage_account, name, session):
 
         queue_service = QueueService(
             account_name=storage_account.name,
-            token_credential=token)
+            token_credential=token,
+            endpoint_suffix=session.endpoints.suffixes.storage_endpoint
```

Comment:
Should we add `StorageEndpoint` property to the `Session` to hide the internals (like `endpoints.suffixes`)